### ISSUE TITLE
[Dropdown] Narrator is not reading the selected value when dropdown is focused

### ISSUE THREAD

**Issue by Bhavana-Nimma** [microsoft/fluentui#19010]
### Environment Information

- **Package version(s)**:  8.21.1
- **Browser and OS versions**:  Internet Explorer (IE11), Microsoft Edge legacy, Windows 10

### Describe the issue:

When the dropdown is focused, narrator is not announcing the selected value.

#### Actual behavior:

When a dropdown is focused, narrator is reading `<aria-label>`, combo box, collapsed, has popup,

![image](https://user-images.githubusercontent.com/85894656/126266205-14143009-33aa-4123-a2bf-1e7743a6d2b2.png)

#### Expected behavior:

Narrator should read `<aria-label>`, `<selected value>`, combo box, collapsed, has popup,

![image](https://user-images.githubusercontent.com/85894656/126266225-56cf28e0-dfe1-4c33-83b9-2153190da71b.png)

**Comment by ling1726** [comment]
Set to low priority for now since this affects IE11 which is reaching end of support for M365 next month, and legacy Edge which we're uncertain what our support level should be.

@smhigley  could you take a look at this from an a11y perspective to see if it's severe enough to move it from low priority ?

@Bhavana-Nimma  thanks for filing this issue could you please also provide a codepen with the repro ?

**Comment by Bhavana-Nimma** [comment]
@ling1726  We are using fluent version 8* but the home page (`https://developer.microsoft.com/en-us/fluentui`) and the examples page of fluent version 8* (`https://fabricweb.z5.web.core.windows.net/oufr/8.21.1/`) are not loading in IE11 whereas the page loads with fluent version 7* (`https://fabricweb.z5.web.core.windows.net/oufr/7.159.0/`). So I am unable to give you the exact code pen repro of fluent 8* Dropdown component in IE11.

P.S: We are able to load our feature pages which uses fluent 8* components and it is when we are facing this narrator issue.

**Comment by ling1726** [comment]
@Bhavana-Nimma  can you fork this codepen: https://codepen.io/ling1726/pen/eYWEzXQ ?

**Comment by Bhavana-Nimma** [comment]
@ling1726 Thanks for the codepen. I have my codepen requirement here - [https://codepen.io/bhavana-nimma/pen/ZEKJBMe](https://codepen.io/bhavana-nimma/pen/ZEKJBMe)

I tested my codepen in Cross Browser testing tool ([https://crossbrowsertesting.com/](https://crossbrowsertesting.com/)). With the help of this tool, we will be able to test in **Edge Legacy** and there we can repro this narrator issue. 
Whereas if I try this codepen on IE11, I am seeing the below screen.

![image](https://user-images.githubusercontent.com/85894656/126322498-d3fbcdd1-4a4c-4eb4-b8ba-fd91c719ae7e.png)

**Comment by ling1726** [comment]
@Bhavana-Nimma thanks for the codepen, leaving as low priority until @smhigley can determine otherwise for a11y cases. IE11 is still a platform that is planned to be moved away soon, and this error does not block products from shipping on the platform.

Edge Legacy is still a question for us tho :/

**Comment by ketaniyengar-msft** [comment]
Hey @smhigley, are there any updates on this bug? Please let me know if you need any help on this.
